qutebrowser/keyinput/keyutils.py

```diff
@@ -71,6 +76,17 @@ def is_modifier_key(key):
     _assert_plain_key(key)
     return key in _MODIFIER_MAP
 
+def _remap_unicode(key, text):
+    # Work around Qt having bad values for higher Unicode characters
+    # Qt events have the upper half of the UTF-16 representation as key()
+    # instead of the unicode codepoint. We re-parse these from text()
+    if _is_extended_unicode(key):
+        if len(text) != 1:
+            raise KeyParseError(text[0], "Key had too many characters!")
```

Comment:
Shouldn't this be `text` instead of `text[0]`?

Reply:
Good catch, thanks

Reply:
@cincodenada Instead of leaving thanks comment you can use +1 and mark it as resolved.